records/hr.record.ts

```diff
@@ -0,0 +1,78 @@
+import { HrEntity } from '../types';
+import { ValidationError } from '../utils/handleErrors';
+import { v4 as uuid } from 'uuid';
+import { FieldPacket } from 'mysql2';
+import { pool } from '../utils/db';
+
+type HrRecordResult = [HrEntity[], FieldPacket[]];
+
+export class HrRecord implements HrEntity {
+  id?: string;
+  user_id: string;
```

Comment:
Dobrze trzymać się jednego zapisu. Skoro `maxReserved...` i `fullName` to także `userId` i `imgSrc`.